scripts/compose.py

```diff
@@ -1178,16 +1187,6 @@ def add_arguments(cls, parser):
             default=cls.DEFAULT_LOCAL_REPO,
         )
 
-    def __init__(self, **options):
-        super(OpbeansPython, self).__init__(**options)
-        self.local_repo = options.get("opbeans_python_local_repo", self.DEFAULT_LOCAL_REPO)
```

Comment:
I removed this special handling, assuming that nobody really needs to test 6.2 with opbeans anymore. If they do, they can use the branch flag manually